### PR TITLE
Bugfix/avoid weird names

### PR DESCRIPTION
**Pull request by AndreeaCh**

### CODE REVIEW

File: src/store/modules/management.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import Function from "../../contracts/management/functions/function";
 import Truck from "../../contracts/management/trucks/truck";
 import SubUnit from "../../contracts/management/subunits/subUnit.js";
 import Utils from "../../services/utils";
+import { stat } from "fs";
```
Comment (mcserby) [inline src/store/modules/management.js]: why this import from filesystem?

File: src/store/modules/management.js [hunk 2]
```diff
@@ -108,6 +117,13 @@ const mutations = {
...
+    } else {
+      let functionsWithId = state.managedFunctions.filter(s => s.id === state.selectedFunction.id);
+      if (functionsWithId.length > 0) {
+        state.selectedFunction = functionsWithId[0];
+      } else {
+        state.selectedFunction = state.managedFunctions[0];
+      }
     }
   },
   [M.SELECT_MANAGED_TRUCK]({ commit }, managedTruck) {
```
Comment (mcserby) [inline src/store/modules/management.js]: the mutator parameter "managedTruck" is never used. it should be removed.

File: src/store/modules/management.js [hunk 3]
```diff
@@ -92,8 +93,16 @@ const mutations = {
...
+    } else {
+      let unitsWithId = state.managedSubUnits.filter(s => s.id === state.selectedSubUnit.id);
+      if (unitsWithId.length > 0) {
+        state.selectedSubUnit = unitsWithId[0];
+      } else {
+        state.selectedSubUnit = state.managedSubUnits[0];
+      }
     }
   },
   [M.SELECT_MANAGED_FUNCTION]({ commit }, managedFunction) {
```
Comment (mcserby) [inline src/store/modules/management.js]: the mutator parameter "managedFunction" is never used. perhaps it should be removed

File: src/store/modules/management.js (hunk 1 above)
Comment (AndreeaCh) [reply src/store/modules/management.js]: dunno. i removed it :)

File: src/store/modules/management.js (hunk 2 above)
Comment (AndreeaCh) [reply src/store/modules/management.js]: but it is used :(

File: src/store/modules/management.js (hunk 3 above)
Comment (AndreeaCh) [reply src/store/modules/management.js]: but it is used :(

File: src/store/modules/management.js (hunk 2 above)
Comment (mcserby) [reply src/store/modules/management.js]: Sorry, my appologies, I should have read the entire file here

**Pull request merged**